app/presenters/users/sessions/reset_password.rb

```diff
@@ -1,27 +1,27 @@
 module Users
   module Sessions
     class ResetPassword < ApplicationPresenter
-      def initialize(view_context, token)
+      def initialize(view_context, token, user)
         @token = token
+        @user = user
         super(view_context)
       end
 
       def page_title
-        "Reset Password | #{school_name}"
+        I18n.t("presenters.users.reset_password", title: current_school.name)
```

Comment:
The key is incorrect; 
```suggestion
     I18n.t('users.sessions.reset_password.page_title', school_name: current_school.name)
```